doc/source/schemas/alleleAnnotations.rst

```diff
@@ -141,14 +141,14 @@ This protocol defines types used by the GA4GH Allele Annotation API.
   :field description:
     A description of the experiment.
   :type description: null|string
-  :field created:
+  :field createTime:
```

Comment:
So: This was ```created``` (ms from epoch) => ```recordCreateTime``` (ISO, my implementation) => ```created``` (ISO, since "created" existed somewhere in the schema) => ```createTime``` (ISO, since ```created``` is very ambiguous).

I am certainly for  ```recordCreateTime```, but flexible (had accepted ```created```, just to get ISO).